### DISCUSSION TITLE
Where can I find the github zsh theme from the light screenshot ?

### DISCUSSION THREAD

**Discussion by MartinDelille** [dandavison/delta#1443] [General]
The "github zsh theme" google search doesn't really give me what I want so I wondered if you could link the theme used in this screenshot:

![github zsh theme](https://user-images.githubusercontent.com/52205/81058911-6abcf700-8e9d-11ea-93be-e212824ec03d.png)

There is not so much light theme and this one looks nice!

**Comment by dandavison** [comment]
Hi @MartinDelille, it's not a Zsh theme. Just set your terminal emulator to have a white background, and set

```
DELTA_THEME = GitHub
BAT_THEME = GitHub
```

and use the `--light` option of delta.

**Comment by dandavison** [comment]
You might also like the "hoopoe" delta theme: see https://dandavison.github.io/delta/custom-themes.html for instructions (it's not just a case of setting an env var).
